tests/tests_data/streaming/test_dataset.py

```diff
@@ -48,15 +49,6 @@ def test_streaming_dataset(tmpdir, monkeypatch):
     assert len(dataloader) == 408
 
 
-@mock.patch.dict(os.environ, {"LIGHTNING_CLUSTER_ID": "123", "LIGHTNING_CLOUD_PROJECT_ID": "456"})
-@mock.patch("lightning.data.streaming.dataset.os.makedirs")
-def test_create_cache_dir_in_lightning_cloud(makedirs_mock, tmpdir):
-    # Locally, we can't actually write to the root filesystem with user privileges, so we need to mock the call
-    with pytest.raises(FileNotFoundError, match="`/cache/chunks` doesn't exist"):
-        StreamingDataset(tmpdir)
-    makedirs_mock.assert_called_once_with("/cache/chunks", exist_ok=True)
```

Comment:
Why are you deleting this? This covers the case where the dir doesn't exist, and I added the fix in #18955

Reply:
reverted.